interface/usergroup/addrbook_list.php

```diff
@@ -76,11 +78,16 @@
 $query .= " LIMIT 500";
 $res = sqlStatement($query, $sqlBindArray);
 ?>
+
+<!DOCTYPE html>
 <html>
 
 <head>
 
-<link rel="stylesheet" href='<?php echo $css_header ?>' type='text/css'>
+<?php Header::setupHeader(['common']); ?>
+<link rel="stylesheet" href="<?php echo $css_header;?>" type="text/css">
```

Comment:
Can remove above line since setupHeader brings it in.

Reply:
I thought so too but wasn't getting style. I'll check again...

Reply:
Now I remember. The load order seems to allow bootstrap to override theme styles font settings. But then and again, some themes are setting font-family and it looks like the classes in the style.css file were taken from scripts and distributed across all the various themes.
With themes still in transition, some latitude will be necessary for some styling in scripts. Because I don't have the patient to add a theme change to 15 different themes.
So in this case the previous classes for the form are no longer needed and I am renaming addressbook_list and leaving in the style.css as it may be a repository for classes needing to get into the theme work flow.
I do not envy those doing the themes...

Reply:
Actually, it's 23 themes :)
And I'm happy to do this stuff (I'll drop include for style.css and bring your changes to all the themes) (in the end, keeping things consolidated is important for maintaining support for RTL and long term sanity).

Reply:
btw, I'll do the above after your work is in the codebase. If you think it's ready to go in, just let me know(If you want more testing, jsut let me know, and I'll assign an up for grabs demo to this).

Reply:
And to clarify, do you need to reinclude $css_header after setupHeader for your styling to work?

Reply:
Can't there be a separate stylesheet for stuff that is common among the themes?

Reply:
There is this:
https://github.com/openemr/openemr/blob/master/interface/themes/core.css

But that is for basic low level common stuff.

What I've noted in watching the themes get improved by folks with different styling goals is that it's best to separate all styling. In the future, goal is to have someting like SASS to support multiple spreadsheets that are similar (for example, just different coloring schemes), but that would also add complexity (for example, would need to compile the SASS to build the theme files when make a change). In the end, simply making modifications to 23 themes is really not tough with phpstorm or sublime(sublime is really cool, because you can actually do the edits in all the files simultaneously; not sure if phpstorm supports that).

Reply:
Okay, don't get me wrong. I thoroughly get the challenge here. SASS is the way to go and I think Robert has this in mind and is close to being there. Transitional problem is we keep supporting the old themes while trying to move forward. Face it, the existing themes(except the light theme) only support bootstrap marginally. Most those classes have to go and in the interim core.css seems to style raw bootstrap tables well with positioning the bugger boo in old themes. Like I say a challenge for sure.
I do like the idea tho of using a style.css for adding new styles for a one stop place when sass is implemented. I guessing that the light style theme will become the base theme for coloring the other themes.
So, for now I am able to get away with not adding any new classes to this form that will affect themes in general and will find some existing classes to get what I need...

Reply:
Themes are definitely in a sort of limbo now as OpenEMR's user interface undergoes drastic improvements. A couple things to review here:
1. The legacy themes were all recently removed.
2. Light was kept and is the default.
3. Tan was changed to Manila since although it was based on legacy it had lots of modernization work done by Ray.
4. 21 new themes were recently brought in by zbig01 as part of an ongoing huge effort to bootstrapify(with responsive and modals) the entire codebase. Currently in progress of integrating all zbig01's work into the codebase; the PR for this is at https://github.com/openemr/openemr/pull/1115 .